config/routes.rb

```diff
@@ -3,11 +3,10 @@
     root to: 'dashboards#index', as: :authenticated_root
     get 'dashboard', to: 'dashboards#index', as: 'dashboard'
     post 'search', to: 'dashboards#search', defaults: { format: :js }
-    resources :pregnancies, only: [ :edit, :update ] do
+    resources :pregnancies, only: [ :create, :edit, :update ] do
```

Comment:
I totally hear you about the messiness, but I think I'm okay with this. After doing a lot of thinking I landed on 'patient and pregnancy should be separate objects, with a patient having a max of one active pregnancy at a time, and any number of inactive pregnancies' as (imo!) the best way to represent the way these pieces fit together. So we don't really have pregnancies existing in isolation in practice.

(Since this body being unearthed is pretty much my fault, I'm happy to make this routing change and associated controller/view changes, haha.)